theme_beauty/views/snippets/s_cover.xml

```diff
@@ -4,7 +4,7 @@
 <template id="s_cover" inherit_id="website.s_cover">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_full_screen_height bg-black-15 o_cc o_cc5" remove="bg-black-50" separator=" "/>
+        <attribute name="class" add="o_full_screen_height bg-black-15 o_cc o_cc5" remove="bg-black-50 o_cc o_cc4" separator=" "/>
```

Comment:
You're both adding and removing the `o_cc` class ... lucky the adding operation is processed after the removal, so the class it's there, but let's fix that ;)

```suggestion
        <attribute name="class" add="o_full_screen_height bg-black-15 o_cc5" remove="bg-black-50 o_cc4" separator=" "/>
```